test/tx-builder/dydx-governance.spec.ts

```diff
@@ -53,6 +58,27 @@ describeContract('DydxGovernance', init, (ctx: TestContext) => {
         );
         expect(emptyVoters.size).to.be.eq(0);
       });
+
+      it('getUserBalancesPerEpoch', async () => {
+        const userBalancesPerEpoch = await txBuilder.merkleDistributorService.getUserBalancesPerEpoch();
+        expect(Object.keys(userBalancesPerEpoch).length).to.be.eq(3);
+        expect(Object.keys(userBalancesPerEpoch['2']).length).to.be.eq(40867);
+      });
```

Comment:
IMO this doesn't need to be tested here, outside of the scope of this PR (already tests in `governance-private`)